src/rpc/misc.cpp

```diff
@@ -243,6 +243,7 @@ static RPCHelpMan deriveaddresses()
             {
                 {"descriptor", RPCArg::Type::STR, RPCArg::Optional::NO, "The descriptor."},
                 {"range", RPCArg::Type::RANGE, RPCArg::Optional::OMITTED_NAMED_ARG, "If a ranged descriptor is used, this specifies the end or the range (in [begin,end] notation) to derive."},
+                {"require_checksum", RPCArg::Type::BOOL, RPCArg::Default{true}, "Fail even if the descriptor checksum is missing. (Always fails if a checksum is present, but invalid.)"},
```

Comment:
"Require a checksum. If a checksum is provided it will be checked regardless of this parameter."